mscore/menus.cpp

```diff
@@ -872,6 +872,16 @@ Palette* MuseScore::newLinesPalette(bool basic)
             pedal->setBeginText("<sym>keyboardPedalPed</sym>");
             pedal->setEndHook(true);
             sp->append(pedal, QT_TRANSLATE_NOOP("Palette", "Pedal"));
+
+            pedal = new Pedal(gscore);
+            pedal->setLen(w);
+            pedal->setBeginText("<sym>keyboardPedalPed</sym>");
+            pedal->setEndText("<sym>keyboardPedalUp</sym>");
```

Comment:
Ah, thanks, I did look but didn't see that colum in the table.  FWIW, the first examples I found don't use continue text, and our other Ped. line doesn't have it either, but I'll add to both.